pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -71,8 +72,8 @@ const (
 const (
 	// BackupFormatDescriptorTrackingVersion added tracking of complete DBs.
 	BackupFormatDescriptorTrackingVersion uint32 = 1
-
-	dateBasedIncFolderName  = "/20060102/150405.00"
+	// DateBasedIncFolderName exported for cliccl backup inspection
```

Comment:
Since we're documenting this here, I think we could also add something like:
```
// DateBasedIncFolderName is the date format used when creating sub-directories
// storing incremental backups for auto-appendable backups.
// It is exported for testing backup inspection tooling.
```